src/components/PatternList.js

```diff
@@ -88,8 +88,10 @@ const PatternList = ({ route, navigation, completedGrammars }) => {
                     title={`Lesson ${grammar.position + 1}`}
                     description={grammar.label}
                     titleNumberOfLines={1}
-                    descriptionStyle={{ fontSize: 20, color: "black" }}
-                    titleStyle={{ fontSize: 16 }}
+                    descriptionStyle={selectedGrammarId === grammar.id
+                      ? { fontSize: 20, color: "#fff" }: { fontSize: 20}}
```

Comment:
Good idea! I think this could be a little hard to read in the future. 
Maybe you can move the styles into `const` and just use the const in the ternary operation? 

like:
```
const defaultStyle = { fontSize: 20, }
const selectedStyle = { ...defaultStyle, color: "#fff" };
```
Then

```
descriptionStyle={selectedGrammarId === grammar.id ? selectedStyle : defaultStyle}
```

What do you think?